.env

```
@@ -0,0 +1,3 @@
+VITE_APP_EMAILJS_SERVICE_ID=service_ehg5hoc
+VITE_APP_EMAILJS_TEMPLATE_ID=template_zil9ryc
+VITE_APP_EMAILJS_PUBLIC_KEY=ty3Ww66ZbbzrXNnuX
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)